File: C#/6kyu - C.cs
//Link -> https://www.codewars.com/kata/5202ef17a402dd033c000009

using System;
using System.Linq;

public class Kata
{
  public static string TitleCase(string title, string minorWords="")
  {    
      if(title != string.Empty)
      {
         string[] words = title.Split();

         if(string.IsNullOrEmpty(minorWords))
         {
             minorWords = "";
         }

         string[] exceptions = minorWords.ToLower().Split();
         string result = string.Empty;
         result += Char.ToUpper(words[0][0]) + words[0].Substring(1).ToLower() + " ";

         foreach (string word in words.Skip(1).ToArray())
         {
            if (exceptions.Contains(word.ToLower()) == false)
            {
               result += Char.ToUpper(word[0]) + word.Substring(1).ToLower() + " ";
            }
            else
            {
               result += word.ToLower() + " ";
            }

          }

          return result.Substring(0, result.Length - 1);
          }
      else
      {
        return title;
      }
  }
}

File: javascript/6kyu - C.js
//Link -> https://www.codewars.com/kata/5848565e273af816fb000449

var encryptThis = function(text) {
    const words = text.split(" ");
    var textEncryption = "";

    for(const word of words){
        var wordSize = word.length;

        switch(wordSize){
            case 1:
                textEncryption += word[0].charCodeAt(0);
                textEncryption += " ";
                break;
            case 2:
                textEncryption += word[0].charCodeAt(0);
                textEncryption += word[1];
                textEncryption += " ";
                break;
            case 3:
                textEncryption += word[0].charCodeAt(0);
                textEncryption += word[2];
                textEncryption += word[1];
                textEncryption += " ";
                break;
            default:
                textEncryption += word[0].charCodeAt(0);
                textEncryption += word[word.length - 1];
                textEncryption += word.substring(2, word.length - 1);
                textEncryption += word[1];
                textEncryption += " ";
        }
    }     

    return textEncryption.trim();
}
